src/tga/decoder.rs

```diff
@@ -261,7 +261,8 @@ impl<R: Read + Seek> TGADecoder<R> {
                     format!(
                         "Color format not supported. Bit depth: {}, Alpha bits: {}",
                         other_channel_bits, num_alpha_bits
-                    ).to_string(),
+                    )
+                    .to_string(),
```

Comment:
Wait, `format!().to_string()`?

Reply:
Hehe, it's odd. `format!()` should return `String`.